Public/Src/Engine/Dll/Engine.cs

```diff
@@ -2409,12 +2409,17 @@ private bool VerifyJournalAvailableForPath(LoggingContext loggingContext, Absolu
         {
             Contract.Requires(path.IsValid);
 
-            var translatedPath = m_translator?.Translate(Context.PathTable, path) ?? path;
-            bool possiblyEnabled = IsJournalPossiblyAvailableForPath(Context.PathTable, translatedPath);
+            // var translatedPath = m_translator?.Translate(Context.PathTable, path) ?? path;
+            bool possiblyEnabled = IsJournalPossiblyAvailableForPath(Context.PathTable, path, out AbsolutePath finalPath);
             if (!possiblyEnabled)
             {
-                var drive = translatedPath.GetRoot(Context.PathTable).ToString(Context.PathTable).TrimEnd('\\');
-                Logger.Log.JournalRequiredOnVolumeError(loggingContext, drive, GetConfigureJournalCommand(drive));
+                var drive = finalPath.GetRoot(Context.PathTable).ToString(Context.PathTable).TrimEnd('\\');
+                Logger.Log.JournalRequiredOnVolumeError(
+                    loggingContext,
+                    drive,
+                    path.ToString(Context.PathTable),
+                    path != finalPath ? finalPath.ToString(Context.PathTable) : string.Empty,

```

Comment:
>string.Empty, [](start = 80, length = 13)

I wonder if the log will be confusing with this empty. Would it be better to just include the final path even when it's the same as the requested path? Or maybe you could have a canned string like "PathIsFinal"